bHY6yh-de4030/cod-yyyiou_bHY6yh-de4030.xml

```diff
@@ -0,0 +1,259 @@
+   <teiHeader>
+      <fileDesc>
+         <titleStmt>
+            <title>Rubrica 15</title>
+            <author ref="#per-c7y65a">Geremia da Montagnone</author>
+            <respStmt>
+               <name ref="#jeffreycwitt">Jeffrey C. Witt</name>
+               <resp>Transcription Editor</resp>
+               <resp>TEI Encoder</resp>
+            </respStmt>
+            <respStmt>
+               <name ref="#alasdairwsliter">Alasdair W. Sliter</name>
+               <resp>Transcription Editor</resp>
+               <resp>TEI Encoder</resp>
+            </respStmt>
+         </titleStmt>
+         <editionStmt>
+            <edition n="0.0.0-dev">
+               <title>Rubrica 15</title>
+               <date when="2023-06-08">June 8, 2023</date>
+            </edition>
+         </editionStmt>
+         <publicationStmt>
+            <authority>SCTA</authority>
+            <availability status="free">
+               <p>Published under a <ref target="https://creativecommons.org/licenses/by-nc-sa/4.0/">Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)</ref>
+               </p>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <listWit>
+               <witness xml:id="V" n="cod-pu6y7d">Vatican, Biblioteca Apostolica Vaticana, Ross. 429 (IX.119)</witness>
+            </listWit>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+         <editorialDecl>
+            <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+       guidelines for a diplomatic edition.</p>
+         </editorialDecl>
+      </encodingDesc>
+      <revisionDesc status="draft">
+         <listChange>
+            <change when="2023-06-08" status="draft" n="0.0.0">
+               <p>Created file for the first time.</p>
+            </change>
+         </listChange>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="la">
+      <front>
+         <div xml:id="starts-on">
+            <pb ed="#C" n="6-v"/>
+            <cb ed="#C" n="a"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-de4030"><!-- p1l1-r15 -->
+            <head xml:id="bHY6yh-de4030-Hd1e108">Rubrica 15</head>
+            <head xml:id="bHY6yh-de4030-Hd1e111" type="question-title">De oratione religiosa</head>
+            <p xml:id="bHY6yh-de4030-d1e114">
+              [omitted]
+            </p>
+            <p xml:id="bHY6yh-de4030-d1e119">
+              <ref xml:id="bHY6yh-de4030-Rd1e116">PSalmista pslamo 33</ref> <quote xml:id="bHY6yh-de4030-Qd1e123">Oculi domini super ius<lb ed="#P" break="no"/>tos 
```

Comment:
Look like replace method might have changed the `#C` to `#P`. Can you change all of these to `#C`

```suggestion
              <ref xml:id="bHY6yh-de4030-Rd1e116">PSalmista pslamo 33</ref> <quote xml:id="bHY6yh-de4030-Qd1e123">Oculi domini super ius<lb ed="#C" break="no"/>tos 
```